requirements.txt

```diff
@@ -4,4 +4,5 @@ lightkube>=0.10.1,<1.0.0
 jsonschema
 ops>=1.3.0,<2.0.0
 pyyaml
-git+https://github.com/canonical/ops-lib-manifest.git@main
\ No newline at end of file
+git+https://github.com/canonical/ops-lib-manifest.git@main
+git+https://github.com/charmed-kubernetes/interface-kube-control.git@ops-package/requires#subdirectory=ops
```

Comment:
THIS CANNOT MERGE!

Reply:
Need this merged first
* https://github.com/juju-solutions/interface-kube-control/pull/36

Reply:
Will need this merged first:
* https://github.com/juju-solutions/interface-kube-control/pull/36